### PR TITLE
Update standard_artifacts.py

### PR DESCRIPTION
**Pull request by anonymousdouble**
refactor with fstring to format string to make code more Pythonic.

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @anonymousdouble Please don't use "add file"/"update file"/"fix file"/etc. commit messages. These are hard to reason about when looking at the history of the file/repository. Instead, please write explanatory git commit messages.

The commit message is also the title of the PR if the PR has only one commit. It is thus twice important to have commit messages that are relevant, as PRs would be easier to understand and easier to analyze in search results.

For how to write good quality git commit messages, please consult https://cbea.ms/git-commit/

Thank you for your contribution!

**Comment by gbaned** [comment]
Hi @anonymousdouble Can you please fix the build [failures](https://github.com/tensorflow/tfx/actions/runs/7384229421/job/20087393096?pr=6546). Thank you!

image

**Review by roseayeon — APPROVED** [review]
The CI Python version has been updated to 3.9. However, unrelated changes are causing the build to fail. As these changes are unrelated, I'll approve this PR.

**Review by gbaned — APPROVED** [review]

File: tfx/types/standard_artifacts.py [hunk 1]
```diff
@@ -363,8 +363,7 @@ class Boolean(ValueArtifact):
 
   def encode(self, value: bool):
     if not isinstance(value, bool):
-      raise TypeError('Expecting bytes but got value %s of type %s' %
-                      (str(value), type(value)))
+      raise TypeError(f'Expecting bytes but got value {str(value)} of type {type(value)}')
```
Comment (chongkong) [inline tfx/types/standard_artifacts.py:366]: `str(x)` is unnecessary; just use `x`. Same for others.

**Review by chongkong — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
Hi @anonymousdouble Can you please check @chongkong's comments and keep us posted ? Thank you!

**Review by roseayeon — APPROVED** [review]

**Pull request merged**